WinGetAutomation/Formula/s/stax76.mpvdotnet.json

```
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://github.com/vedantmgoyal9/vedantmgoyal9/raw/main/WinGetAutomation/schema.json",
+  "PackageIdentifier": "stax76.mpvdotnet",
```

```suggestion
  "PackageIdentifier": "mpvnet-player.mpvnet",
```

microsoft/winget-pkgs#152192